tensorflow/python/distribute/distribute_lib.py

```diff
@@ -959,6 +960,7 @@ def make_input_fn_iterator(self,
           input_fn, replication_mode=replication_mode)
 
   @doc_controls.do_not_generate_docs  # DEPRECATED: TF 1.x only
+  @deprecated(None,'use mirrored_strategy.run() instead')  
```

Comment:
use run() instead.

Reply:
Made the required change at line 963. Let me know if any other changes are required.